sites/dotnet-guide/contents/authors/pwe/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Paul Everitt
+subtitle: PyCharm Developer Advocate at JetBrains
+thumbnail: ./pwe.jpg
+---
+
+Paul is the PyCharm Developer Advocate at JetBrains. Before that, Paul
```

Comment:
```suggestion
Paul is a member of the PyCharm Developer Advocates at JetBrains. Before that, Paul
```

Reply:
Up to @pauleveritt , as this may need other Guides to update as well. Paul, "commit suggestion" at will.